virtualization/windowscontainers/management/container_networking.md

```diff
@@ -214,8 +216,37 @@ C:\> docker network inspect <network name>
 ### Multiple Container Networks
 
 Multiple container networks can be created on a single container host with the following caveats:
-* Only one NAT network can be created per container host.
+
 * Multiple networks which use an external vSwitch for connectivity (e.g. Transparent, L2 Bridge, L2 Transparent) must each use its own network adapter.
+* Currently, our solution for creating multiple NAT networks on a single container host is to subnet the NAT network IP address block. Reference the below section, 'Multiple NAT Networks' for further guidance on this.
+
+### Multiple NAT Networks
+It is possible to define multiple NAT networks on a single container host by subnetting the host's NAT network address block. 
+
+The IP address block prefix for the container host's NAT network can be found using the below command.
+
+```none
+PS C:\> get-netnat
+```
+> Note: The container host's NAT network prefix can be changed by removing and recreating the network with the desired prefix.
+```none
+PS C:\> remove-netnat
+PS C:\> new-netnat -name nat  -InternalIPInterfaceAddressPrefix <NAT IP Prefix>
+```
+For example, the NAT network prefix might be the default that was created by the Windows Docker engine, 172.16.0.0/12. In this case, Docker can be used to create additional NAT networks as long as they fall under the 172.16.0.0/12 prefix.
+
+For example, two NAT networks could be created with the IP prefixes 172.16.0.0/16 (gateway, 172.16.0.1) and 172.17.0.0/16 (gateway, 172.17.0.1). 
```

Comment:
These two subnets overlap which is not quite what we want. Instead, if the existing NAT IP prefix is 172.16.0.0/16, we could create a new nat network as partition of this existing prefix. e.g. 172.17.0.0/24. 

172.16.0.0/16 includes all subnets in the range 172.16.0.0 - 172.16.255.0